openquake/hazardlib/gsim/chiou_youngs_2014.py

```diff
@@ -178,7 +178,7 @@ def _get_ln_y_ref(self, rup, dists, C):
         mag_test1 = np.cosh(2. * max(rup.mag - 4.5, 0))
 
         # centered DPP
-        centered_dpp = 0.
+        centered_dpp = self._get_centered_cdpp(dists)
```

Comment:
the cdpp will only be computed in ChiouYoungs2014NearFaultEffect.

The _get_centered_cdpp method is different in ChiouYoungs2014 (without directivity, the cdpp = 0) and ChiouYoungs2014NearFaultEffect( cdpp is called from  dists.rcdpp) 

ChiouYoungs2014